tdrs-backend/tdpservice/email/helpers/data_file.py

```diff
@@ -1,25 +1,71 @@
 """Helper functions for sending data file submission emails."""
+from django.conf import settings
 from tdpservice.email.email_enums import EmailType
 from tdpservice.email.email import automated_email, log
+from tdpservice.parsers.util import get_prog_from_section
 
 
-def send_data_submitted_email(recipients, data_file, context, subject):
-    """Send an email to a user when their data has been submitted."""
-    template_path = EmailType.DATA_SUBMITTED.value
-    text_message = 'Your data has been submitted.'
+def send_data_submitted_email(
+    datafile_summary,
+    recipients,
+):
+    """Send an email to a user when their account approval status is updated."""
+    from tdpservice.parsers.models import DataFileSummary
+
+    datafile = datafile_summary.datafile
 
     logger_context = {
-        'user_id': data_file.user.id,
-        'object_id': data_file.id,
-        'object_repr': f"Uploaded data file for quarter: {data_file.fiscal_year}"
+        'user_id': datafile.user.id,
+        'object_id': datafile.id,
+        'object_repr': f"Uploaded data file for quarter: {datafile.fiscal_year}"
+    }
+
+    template_path = None
+    subject = None
+    text_message = None
+
+    section_name = datafile.section
+    prog_type = get_prog_from_section(section_name)
+    file_type = f'{prog_type}F' if prog_type != 'SSP' else prog_type
```

Comment:
@jtimpe thank you for this!

![tanf_spelling](https://github.com/raft-tech/TANF-app/assets/63075587/5b2496cc-7b37-4c4b-9221-3499686a9cca)